### PR TITLE
16# Resolve conflicts

### PR DESCRIPTION
**Pull request by abalanag**
Resolved conflicts between the differet versions of Bootstrap used for header and homepage
Created Thymeleaf links for all the CSS files.

### CODE REVIEW

File: src/main/resources/templates/header.html [hunk 1]
```diff
@@ -1,98 +1,105 @@
 <!doctype html>
 
-<html xmlns:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" th:fragment="header-info">
```
Comment (danutchindris) [inline src/main/resources/templates/header.html]: I don't think this is a good idea. Are you going to inject the `<html>` element inside another full HTML document? Doesn't sound right. You should inject the `<header>` element or something like that.

**Review by danutchindris — CHANGES_REQUESTED** [review]
Please see the review comments.

File: src/main/resources/templates/header.html (hunk 1 above)
Comment (abalanag) [reply src/main/resources/templates/header.html]: Done :)

**Review by danutchindris — APPROVED** [review]
Looks OK.

**Pull request merged**